Parkinson/Startup/APHAppDelegate.m

```diff
@@ -39,7 +39,13 @@
 static NSString *const kVoiceActivitySurveyIdentifier               = @"3-APHPhonation-C614A231-A7B7-4173-BDC8-098309354292";
 static NSString *const kTappingActivitySurveyIdentifier             = @"2-APHIntervalTapping-7259AC18-D711-47A6-ADBD-6CFCECDED1DF";
 static NSString *const kMemoryActivitySurveyIdentifier              = @"7-APHSpatialSpanMemory-4A04F3D0-AC05-11E4-AB27-0800200C9A66";
-static NSString *const kMyThoughtsSurveyIdentifier                  = @"8-MyThoughts-12ffde40-1551-4b48-aae2-8fef38d61b61";
+static NSString *const kMyThoughtsSurveyIdentifier                  = @"mythoughts";
+static NSString *const kPDRatingScaleSurveyIdentifier               = @"MDSUPDRS";
+static NSString *const kEnrollmentSurveyIdentifier                  = @"EnrollmentSurvey";
+static NSString *const kPDQuestionnaireSurveyIdentifier             = @"PDQ8";
+static NSString *const kStudyFeedbackSurveyIdentifier               = @"study_feedback";
+
```

Comment:
Looks good, but the PD Rating Scale and PD Questionnaire are copyrighted surveys so we don't want them in the open source codebase--can you take just those two out here, and I'll add them back in after pulling into the AppStore version?